test_package_validation.py

```
@@ -0,0 +1,254 @@
+    else:
+        print("❌ Version not found or incorrect")
+        return False
+    
+    # Check if version is accessible from package
+    try:
+        import zoho_mcp
+        if hasattr(zoho_mcp, '__version__'):
+            print(f"✅ Package version accessible: {zoho_mcp.__version__}")
+        else:
+            print("⚠️  Package version not accessible via __version__ (optional)")
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)